pages/order-list/index.js

```diff
@@ -1 +1,20 @@
-export default function OrderList() {}
+import Drinks from "../components/order-menu/drink-menu";
+import Maindishes from "../components/order-menu/main-menu";
+
+export default function OrderList() {
+  return (
+    <>
+      <h1>Order Menu</h1>
+      <form>
+        <h3>Hauptspeisen</h3>
+        <ul>
```

Comment:
In this case (since the Maindishes is a list of maindishes) I recommend to render the ``<ul>`` in that component.

Reply:
Look at my comment above. There is no <ul> in the Drinks oder Maindishes component